packages/python-debian/python-debian.spec

```diff
@@ -6,8 +6,8 @@
 %global srcname debian
 
 Name:           %{?scl_prefix}python-%{srcname}
-Version:        0.1.42
-Release:        1%{?dist}
+Version:        0.1.44
+Release:        2%{?dist}
```

Comment:
Does it matter that this will result in a package version, that is higher than that built for the `rpm/3.18` branch? (`rpm/3.18` currently uses `0.1.44-1`)

Reply:
We will need to bump again on 3.18 to avoid upgrade problems, having 2 python versions cause all sort of fun problems.

Reply:
Should I go ahead and open a PR for that? Should it bump the `Release` to 3?

Reply:
Feel free to open that one, and we can merge this one after that

Reply:
rpm/3.18 PR: https://github.com/theforeman/pulpcore-packaging/pull/551